### DISCUSSION TITLE
UI Bug when creating store over API

### DISCUSSION THREAD

**Discussion by jon4hz** [btcpayserver/btcpayserver#2433] [Technical Support] [answered]
Hi, I encountered a bug when I created a new store using the greenfield api v1.
Normally BTCPay marks stores with a warning sign, if no wallet is configured. However, if you create it over the API, that sign doesn't appear.

![image](https://user-images.githubusercontent.com/26183582/113633228-d6d74a80-966c-11eb-8d18-2db028aedd3f.png)

In this the stores have the following configuration:
- test01: created over the gui and has a wallet configured
- test02: created over the gui but has no wallet configured
- test03: created by using the api and has no wallet configured

**Comment by NicolasDorier** [comment, marked as answer]
This should be an issue, not a discussion. Can you open one?

**Reply by NicolasDorier** [reply]
Ping @rockstardev

**Reply by Zaxounette** [reply]
I sent him here instead of the issues because pav's process is to confirm a bug before opening an issue.

**Reply by Kukks** [reply]
Yeah, it is just unfortunate we can't convert directly into an issue after confirmation..

**Reply by rockstardev** [reply]
Opened issue #2445 because Emperor @NicolasDorier commanded me with his ping.

**Reply by Zaxounette** [reply]
> Opened issue #2445 because Emperor @NicolasDorier commanded me with his ping.

Issue #2434 was opened by OP and PR #2435 closed it.
